src/config.js

```diff
@@ -0,0 +1,45 @@
+export const DEBUG = false;
+export const GENERAL_ATTRIBUTES = ["Hash", "Year", "CommitDate", "isRelease"];
+export const TEST_VARIABLES = ["Number of Logs", "TPS"];
+export const IGNORE_ATTRIBUTES = [
+  ...GENERAL_ATTRIBUTES,
+  "Link",
+  "Data",
+  "TestID",
+];
+//Metric Specific Configs
+export const UNITS = {
+  procstat_cpu_usage: "%",
+  procstat_memory_rss: "MB",
+};
+export const MetricConfigs = ["thresholds"];
+const N_METRIC = 2; //number of metrics
+const N_TIMESTAMPS = 3; // number of timestamps per metric
+export const N_STATS = 4;
+//CALCULATED CONST
+// 1 commit is # kb, so to make max 1MB batches our batches should be
+const ONE_MB = 1000; //KB
+const BASE_PACKET_SIZE = 0.057; // KB per packet with no timstamp or metric
```

Comment:
This is bound to change. What's the purpose of this?

Reply:
I wanted to draw batches in 1 MB chunks, and this calculation was sufficient before the multiple logs.  If you prefer it too, I think it is best if I just make it one const, and not to the calculations